_posts/en/newsletters/2021-07-28-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #159'
+permalink: /en/newsletters/2021/07/28/
+name: 2021-07-28-newsletter
+slug: 2021-07-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter includes our regular sections with the best
+questions and answers of the past month from the Bitcoin Stack Exchange,
+our latest column about preparing for taproot, a list of new software
+releases and release candidates, and descriptions of notable changes to
+popular Bitcoin infrastructure software.
+
+## News
+
+*No significant news this week.*
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is this unusual transaction in the Bitcoin blockchain?]({{bse}}107603)
+  Murch describes an output labeled "UNKNOWN" in a [block explorer][topic
+  block explorers]. The output is a segwit version 1 output with a contrived
+  pubkey. As pointed out 0xb10c, the 2019 transaction creating this output was for
+  purposes of testing segwit v1 support for [Optech's Compatibility
+  Matrix][compat matrix]. As warned previously (see [Newsletter #158][news158
+  taproot]), 0xb10c also inspects some [anyone-can-spend P2TR outputs][0xB10C blog].
```

Comment:
This sounds as if we warned that 0xb10c would inspect P2TR outputs, but we warned that anyone could spend them before the activation of taproot.

```suggestion
  Matrix][compat matrix]. As warned previously (see [Newsletter #158][news158
  taproot]), P2TR outputs are anyone-can-spend before the activation of taproot as 0xb10c demonstrated and [elaborates in a blog post][0xB10C blog].
```